packages/plugin-core/src/test/suite-integ/GotoNote.test.ts

```diff
@@ -131,6 +131,29 @@ suite("GotoNote", function () {
       });
     });
 
+    test("go to new note with template in a different vault", (done) => {
+      runLegacyMultiWorkspaceTest({
```

Comment:
Recommend avoiding the deprecated harness going forward, even if the other tests are using it. Refactoring this for the new harness would look like this:
```ts
    describeMultiWS(
      "GIVEN a new note and a template in different vaults",
      {
        ctx,
        preSetupHook: ENGINE_HOOKS_MULTI.setupBasicMulti,
        postSetupHook: async ({ wsRoot, vaults }) => {
          await ENGINE_HOOKS.setupSchemaPreseet({ wsRoot, vaults });
        },
      },
      () => {
        test("THEN new note uses that template", async () => {
          // Note is in different vault from template
          const { vaults } = ExtensionProvider.getDWorkspace();
          const vault = vaults[1];
          await createGoToNoteCmd().run({
            qs: "bar.ch1",
            vault,
          });
          expect(getActiveEditorBasename()).toEqual("bar.ch1.md");
          const content =
            VSCodeUtils.getActiveTextEditor()?.document.getText() as string;
          expect(content.indexOf("ch1 template") >= 0).toBeTruthy();
        });
      }
    );
```

Reply:
Sounds good, thanks for suggestion